qa/tests/build-jts.xml

```diff
@@ -87,6 +88,7 @@
             <property name="idl.params2" value="include='${org.jboss.jbossts.qa.orb.home}/${orb.idl.path};${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/idl',classpath='${org.jboss.jbossts.qa.orb.classpath}'"/>
 
             <property name="is.sunorb" value="1"/>
+			<property name="orb.jar.file.name" value="glassfish-corba-omgapi.jar"/>
```

Comment:
I think we need to remove sunorb references for any JDK over JDK 11: http://openjdk.java.net/jeps/320. It wouldn't be the same way we dealt with IBM orb as it is definitely gone now.

Reply:
I see, so I will revert these changes and the JIRA `JBTM-3601` will take care of removal of this `ORB`